src/styles/components/_icon.scss

```diff
@@ -0,0 +1,18 @@
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+
+  &--phone {
+    background-image: url(../images/icons/icon-phone.svg);
+    &:hover + .header__phones {
+      opacity: 1;
+    }
+  }
```

Comment:
At small resolutions your hover works like that,fix it
![image](https://github.com/mate-academy/layout_landing-page/assets/143347271/a3b7dba0-9731-4147-a8f6-8933845e8a21)

Reply:
It should not appear on the phone resolution, fixed that